api/representation_test.go

```diff
@@ -132,7 +134,8 @@ func TestTaskRequest_ToRequestTaskConfig(t *testing.T) {
 				Enabled: config.Bool(true),
 				Condition: &oapigen.Condition{
 					Services: &oapigen.ServicesCondition{
-						Names: &[]string{"api", "web"},
+						Names:             &[]string{"api", "web"},
+						SourceIncludesVar: config.Bool(false),
```

Comment:
Can you change one of theses tests to test the default behavior for `true`?

Reply:
I can change one of the test cases to test when the request has SourceIncludesVar = true, then the config should have SourceIncludesVar = true. Is this what you mean?

Asking since I'm not sure if you might mean the case where the request does not have SourceIncludesVar set, then the config defaults to SourceIncludesVar = true. Since it looks like `ToTaskConfig` no longer calls Finalize() and Validate(), we can't assigning default values here (i.e. the config still has SourceIncludesVar = nil). Let me know if I'm misunderstanding you.

Reply:
Oh I see, this is just the conversion method and the defaults aren't handled here. Nah it's ok! Thank you for your explanation.